lein-environ/test/lein_environ/plugin_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns lein-environ.plugin-test
+  (:require [clojure.test :refer :all]
+            [lein-environ.plugin :as l]))
+
+(deftest replace-project-keyword-test
+  (are [expected value] (= (#'l/replace-project-keyword value {:version "1.0.0"
```

Comment:
If you're going to add a test, it needs to be for public functions. The private functions of a namespace should be able to be changed without affecting the tests.